chpl/chpl-service/src/main/java/gov/healthit/chpl/dto/statistics/PrivacyAndSecurityListingStatisticDTO.java

```diff
@@ -0,0 +1,19 @@
+package gov.healthit.chpl.dto.statistics;
+
+import java.time.LocalDate;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class PrivacyAndSecurityListingStatisticDTO {
```

Comment:
Had we talked about not using DTOs in favor of just using Entities, or something along those lines? As a general case? If not, or I'm misremembering things, that's fine. But if we had, is that something we'd want to do here?

Reply:
So, we had talked about not using DTOs in favor of just using Entities and Domain objects (cutting out the middleman DTO that rarely ever does anything). I chose to make DTOs here thinking that none of these objects are getting sent back to the client in any sort of API call so I didn't feel like they were really "domain" objects. I could be convinced otherwise? Maybe @tmy1313  could weigh in?

Reply:
So if we use domain objects for things that go across the API boundary, and map domain objects to entities (or trend that way) then I think it'd be reasonable to use DTOs as containers for stuff that never goes across the API. I'm good with that, but also would defer to strong opinions in any other direction